### ISSUE TITLE
Scala view don't show available computer

### ISSUE THREAD

**Issue by CarlShowalter** [marcisme/sketch-preview#47]
Hello.
I am using Sketch 49.3, scala view just installed as sketch preview plugin and nothing is working. On android it always shows no computer available. Also sketch show failed to open error when select any artboard.
How to fix this?

**Comment by marcisme** [comment]
Is Skala Preview installed on your mac? That will need to be installed, and you may want to try to use it independently to make sure it’s configured correctly before using it via the plugin.

**Comment by CarlShowalter** [comment]
Yes I had installed Skala Preview on Mac. Now I am using Linux Ubuntu 18.04. I had a problem with sound on my Hakintosh so in an attempt to fix it I broke whole system it didn't want to boot and there is no chance to make sound work on my PC, so I needed to install Linux again (I had MacOs only two days and I installed it only because of Sketch and Scala).

Linux is awesome OS but since there is no Sketch on Linux we have Inkscape, Krita, Gimp and very similar to Sketch relatively new web based cross platform app called Gravit Designer. https://designer.io/

Designers on Linux really need the way to preview designs on Android like with Scala. Is there any chance to make it available for Gravit or open source apps like Inkscape and Krita??

For now Gravit is less powerful than Inkscape or Krita but it is very good for UI designing, have almost everything and it will have prototyping possibilities soon. Inkscape is very powerful vector software but it is not very suitable for UI designing and prototyping and Krita just landed with .svg1 support so it is also promising app for UI designing.

There is one open source service for connecting Linux with Android called KDE connect it works really good. Maybe that can be useful for this. https://community.kde.org/KDEConnect
Android app https://play.google.com/store/apps/details?id=org.kde.kdeconnect_tp
Gnome shell plugin https://github.com/andyholmes/gnome-shell-extension-gsconnect

There is nothing like Scala for Linux and I can't explain you how much we need it..
